doc/rst/developer/bestPractices/StandardModuleStyle.rst

```diff
@@ -130,7 +135,50 @@ Accessor methods will avoid using "get" in their name.  E.g., instead of
 
 Methods that are not accessors are still allowed to use "get" in their name.
 
+Formals
++++++++
+
+Formal names should be camelCase.  Descriptive names are recommended, within
+reason.  Encoding the type name into a formal name is generally avoided when
+multiple types are supported, as doing so makes it harder to support generic
+functions.
+
+Some commonly used names are:
+
+- idx for an index
+
+- x and y for general pairs of formals
+
+- src and dst/dest (source and destination) for directional pairs of formals
+
+  - lhs and rhs (left hand side and right hand side) may also be used
+
+- eltType for the type of elements stored in a collection (also used as a field
+  name)
+
+- obj for a general object instance (as opposed to formals that can also be
+  primitive types)
+
 Other Identifiers
 -----------------
 
-Variables, fields, and argument names should be camelCase or PascalCase.
+Variables and fields should be camelCase or PascalCase.
+
+Handling Failure
+----------------
+
+In general, when code in a library encounters erroneous behavior, an appropriate
+Error should be thrown.  This will enable users of the library to potentially
+recover from the bad behavior without having to restart their program.
+
+There are some known exceptions to this recommendation:
+
+- When division by zero is encountered, libraries will halt (unless the check
+  for division by zero has been disabled, see the flag's
+  :ref:`man page entry <man-div-by-zero-checks>`).
+- When accesses are made outside of the known bounds of a container type,
+  libraries will halt (unless bounds checking has been disabled, see the flag's
+  :ref:`man page entry <man-bounds-checks>`).
+- Other checks controlled by flags that are listed when running ``chpl -h``.
+- When the program runs out of memory and the function in question does not
+  already throw other types of errors, the library will halt.
```

Comment:
Does this bulleted list render right without the spacing between bullets?

Reply:
Yup, I checked the built docs (and just checked again now).  The main important thing is that:
- the indentation level is the same, and that
- the list is separated from other text.

If line 174 had been immediately before line 176 instead of having the space, it would have rendered badly.  Similarly, if we had a sub-bullet like line 154, that would have messed with things.

One could argue that adding spaces will avoid the possibility of us introducing those sorts of errors in the future, so I would be up for adding those line separators anyways if you thought that was a good idea

Reply:
I think yes you should add the, if only to be consistent. The other bullet list you added has the spacing (which is why I noticed), and I never remember which is "the right way" when writing sphinx docs